CPPLogger/CPPLogger.cpp

```diff
@@ -6,14 +6,15 @@
 #include "framework.h"
 #include "CPPLogger.h"
 
-const wchar_t* CLIENT_INIT_PARAM = L"/P7.Sink=FileTxt /P7.Dir=C:\\Program Files\\Logs\\ /P7.Format=\"%ti - %tf [%lv] - [%fs] [%fn] %ms\"";
+const wchar_t* CLIENT_INIT_PARAM = L"/P7.Sink=FileTxt /P7.Dir=C:\\Logs\\ /P7.Format=\"%ti - %tf [%lv] - [%fs] [%fn] %ms\"";
```

Comment:
It need to add option to read log file path from xml file

Reply:
That is not my file. It's CPPLogger.